docs/pages/versions/unversioned/sdk/screen-orientation.md

```diff
@@ -21,15 +21,32 @@ On both iOS and Android platforms, changes to the screen orientation will overri
 
 This API is pre-installed in [managed](../../introduction/managed-vs-bare/#managed-workflow) apps. It is not yet available for [bare](../../introduction/managed-vs-bare/#bare-workflow) React Native apps.
 
+### Warning
+
+This module needs additional configuration to work properly on iPads. Unfortunately, it will disable support for the split view.
```

Comment:
Could you elaborate a bit on what does it mean to _work properly on iPads_, why doesn't this module _work properly_ by default, what are the advantages and disadvantages of adding `requireFullScreen: true` property?